pvextractor/gui.py

```diff
@@ -243,16 +242,26 @@ def __init__(self, filename_or_cube, backend="Qt5Agg", clim=None):
             if self.array.ndim != 3:
                 raise ValueError("dataset does not have 3 dimensions (install the spectral_cube package to avoid this error)")
 
-        self.backend = backend
-
         import matplotlib as mpl
 
-        mpl.use(self.backend)
+        # We reset the rc parameters to the default values to make sure we use
+        # the default interactive backend and to make sure that the UI is
+        # consistent.
+        mpl.rcdefaults()
+
+        if backend is not None:
+            mpl.use(backend)
+
         import matplotlib.pyplot as plt
 
-        self.fig = plt.figure(figsize=(14, 8))
+        self.fig = plt.figure(figsize=(8, 5))
+
+        self.backend = mpl.get_backend()
+        print("Using Matplotlib backend: {0}".format(self.backend))
```

Comment:
should this be a logger call?  Doesn't matter that much, just wondering aloud

Reply:
Fixed